pkg/config/config.go

```diff
@@ -11,4 +11,12 @@ const (
 	Debug string = "debug"
 	// ClusterID represents the ID of the cluster used for OCM notifications
 	ClusterID string = "cluster-id"
+	// FleetMode represents if ocm-agent is going to run in default OSD/ROSA mode or HyperShift mode
+	FleetMode string = "fleet-mode"
+	// TestFleetMode represents if ocm-agent is being run locally simulating fleet-mode run
+	TestFleetMode string = "test-fleet-mode"
```

Comment:
This is left over?

Reply:
Oh yes, that's a miss from my end. I will remove it.